pharus/interface.py

```diff
@@ -219,84 +194,87 @@ def get_table_attributes(jwt_payload: dict, schema_name: str, table_name: str) -
         :type schema_name: str
         :param table_name: Table name under the given schema; must be in camel case
         :type table_name: str
-        :return: Dict with keys ``primary_attributes``, ``secondary_attributes`` containing a
+        :return: Dict with keys ``attribute_headers`` and ``attributes`` containing
+            ``primary_attributes``, ``secondary_attributes`` which each contain a
```

Comment:
I think this should be primary and secondary without the attributes

Reply:
@Synicix Not near my machine right now but if you will propose the change I can accept it.

Reply:
Suggestion added

Reply:
@Synicix Thanks, approved and included it.